pages/apis/graphql_api.md.erb

```diff
@@ -121,4 +121,5 @@ The Buildkite GraphQL API adheres to the [Relay Specification](https://relay.dev
 Further resources for learning more about GraphQL:
 
 * Our [Getting Started with GraphQL Queries and Mutations](https://buildkite.com/blog/getting-started-with-graphql-queries-and-mutations) tutorial
+* Our [GraphQL API Cookbook](/docs/apis/graphql/graphql_cookbook) page full of common queries and mutations
```

Comment:
I wasn't able to test this yet, but I bet its kebab case like the rest
```suggestion
* Our [GraphQL API Cookbook](/docs/apis/graphql/graphql-cookbook) page full of common queries and mutations
```

Reply:
Yes, this should work! We have some info on hyphens and underscores in the [style guide](https://github.com/buildkite/docs/blob/main/styleguide/STYLE.md#adding-and-naming-new-documentation-pages).